src/peergos/shared/NetworkAccess.java

```diff
@@ -37,6 +37,8 @@
     public final MutableTree tree;
     public final WriteSynchronizer synchronizer;
     public final InstanceAdmin instanceAdmin;
+    public final SharedWithCache sharedWithCache;
```

Comment:
This has sensitive information so probably shouldn't be on network access, can it be on usercontext?